src/resources/audit/AuditList.tsx

```diff
@@ -251,7 +284,18 @@ export default function AuditList({
           label='Security Related'
           looseValue
         />
-        <TextField<Audit> source='resource' label='Resource' />
+        <FunctionField<Audit>
+          source='resource'
+          label='Resource'
+          render={(record) => (
+            <span>
```

Comment:
@TahaKhanAbdalli  - have a linter warning on the next line.
![image](https://github.com/DeepBlueCLtd/RCO/assets/1108513/02431252-ebe8-4f5f-ab05-5919a617309e)

Is it worth refactoring it into an arrow function, where we can be a bit more verbose?

Reply:
Let me check.